jujugui/static/gui/src/app/components/login/login.js

```diff
@@ -25,7 +25,8 @@ YUI.add('login-component', function() {
     propTypes: {
       errorMessage: React.PropTypes.string,
       isLegacyJuju: React.PropTypes.bool.isRequired,
-      loginToAPIs: React.PropTypes.func.isRequired
+      loginToAPIs: React.PropTypes.func.isRequired,
```

Comment:
Maybe we should rename the two callables like loginWithUserPass and loginWithUSSO, so that's more clear how they will be used in the component?